PyHa/IsoAutio.py

```diff
@@ -990,7 +990,7 @@ def generate_automated_labels_microfaune(
             if annotations.empty:
                 annotations = new_entry
             else:
-                annotations = annotations.append(new_entry)
+                annotations = pd.concat([annotations, new_entry])
```

Comment:
``So for concat, its a bit slower if we append each row to a dataframe. According to the docs its slightly better to instead save each new_entry to a list then concat the list.

So basically 

list = []
for i in etc:
       list.append(some new row for a future df)
df = pd.concat(list)

Reply:
Got it. TBH I was just doing a find and replace for this PR, didn't fully look at all the context. I'll go back and make the necessary changes